fix(readme): add instructions to run unit and e2e tests

lorumic opened the pull request:
Added instructions to the README of the repo, to explain how to run unit tests and end-to-end tests.

keirthana commented on README.md:
```diff
@@ -25,6 +25,31 @@ Refer to the following examples to explore advanced usage scenarios and best pra
+- Unit tests are located in `js/tests/unit`.
+- End-to-end tests are located in `js/tests/e2e`.
+
+Both these folders contain a `run-tests.sh` script. You just need to run this script to start the tests. However, to run end-to-end tests you need also an Anbox Cloud back-end. The easiest way to set up one is by using the [Anbox Cloud Appliance](https://documentation.ubuntu.com/anbox-cloud/en/latest/tutorial/installing-appliance/).
```
```suggestion
Both these folders contain a `run-tests.sh` script that you can run to start the tests. However, to run end-to-end tests, you also need an Anbox Cloud deployment. The easiest way to set up one is by using the [Anbox Cloud Appliance](https://documentation.ubuntu.com/anbox-cloud/en/latest/tutorial/installing-appliance/).
```

keirthana commented on README.md:
```diff
@@ -25,6 +25,31 @@ Refer to the following examples to explore advanced usage scenarios and best pra
+
+Both these folders contain a `run-tests.sh` script. You just need to run this script to start the tests. However, to run end-to-end tests you need also an Anbox Cloud back-end. The easiest way to set up one is by using the [Anbox Cloud Appliance](https://documentation.ubuntu.com/anbox-cloud/en/latest/tutorial/installing-appliance/).
+
+Once the back-end is ready, follow these steps to run the end-to-end tests:
```
```suggestion
Once the appliance is ready, follow these steps to run the end-to-end tests:
```

keirthana reviewed (changes requested):
Some minor comments

lorumic commented:
Thanks for the great suggestions @keirthana - I have addressed your comments [here](https://github.com/canonical/anbox-streaming-sdk/compare/57bec1d9c35c8852eda6d5d7ced374f9bc8d3970..75a5505971419f199fe7150aad464371f34ed4f7).

morphis reviewed (approved):
LGTM

adglkh reviewed (approved):
LGTM

ajanon reviewed (approved):
LGTM

keirthana reviewed (approved):
Thanks, LGTM
